site/src/theme/mui.ts

```diff
@@ -79,13 +79,13 @@ export let dark = createTheme({
     fontFamily: BODY_FONT_FAMILY,
 
     body1: {
-      fontSize: "1rem" /* 16px at default scaling */,
-      lineHeight: "1.5rem" /* 24px at default scaling */,
+      fontSize: 16,
```

Comment:
My concern with getting rid of rem units is that if someone has a custom base font size defined for their browser (whether it's for accessibility reasons, or just because they like the bigger text), the Coder website will suddenly stop respecting that

I think moving away from rem values is good for the line height, but I feel like the base font sizes should stay the same (even if it does make body2's font size uglier)

Reply:
Sounds fair.